### PR TITLE
Match search and revive FourOhFour

### PR DESCRIPTION
**Pull request by Argn0**

### CODE REVIEW

File: src/actions/action.js [hunk 1]
```diff
@@ -11,17 +11,30 @@ export default function action(type, host, path, params = {}, transform) {
...
+      error,
+    });
     const fetchDataWithRetry = delay => fetch(url, path === 'api/metadata' ? { credentials: 'include' } : {})
       .then((response) => {
         if (!response.ok) {
-          throw new Error('fetch failed');
+          dispatch(getError(response.status));
+          if (response.status === 404) {
+            const err = new Error('api returned 404');
+            err.code = 404;
```
Comment (howardchung) [inline src/actions/action.js]: probably err.code = response.status.  Technically all 4xx codes indicate user error and should not be retried.

File: src/lang/en-US.json [hunk 2]
```diff
@@ -620,6 +621,7 @@
   "rune_6": "Arcane",
 
   "search_title": "Player Search",
+  "search_title_2": "Player, Match Search",
```
Comment (howardchung) [inline src/lang/en-US.json]: probably "Search by player name, match ID..."

Comment (howardchung) [reply src/lang/en-US.json]: Any reason you opted to make a new string instead of replacing the existing one?

**Comment by Argn0** [comment]
-do not retry on any 4xx client error
-fixed hint text

**Pull request merged**